src/lib/mongoBackend/mongoDiscoverContextAvailability.cpp

```diff
@@ -74,7 +74,8 @@ static HttpStatusCode processDiscoverContextAvailability
                           &count))
   {
     responseP->errorCode.fill(SccReceiverInternalError, err);
-    alarmMgr.dbError(responseP->errorCode.details);
+    // FIXME P10: overloggin (registrationQuery() false return menas that collectionRangedQuery() retursn false, corresponding to internal alarms in that function)
```

Comment:
In this case, I think that low level. Along the PR you would see that doing so (i..e removing at high level) makes code simpler.